packages/orchestration/src/utils/mockChainInfo.js

```diff
@@ -1,5 +1,10 @@
 /**
- * @file Mocked Chain Info object until #8879
+ * @file Mocked Chain Info object
```

Comment:
can we move this out of src/ yet?

Reply:
I gave it a try but will save it for a next PR because it affects the a3p test too.